pages/model/[uuid].tsx

```diff
@@ -86,10 +87,15 @@ function Model() {
   const { versions, isVersionsLoading, isVersionsError } = useGetModelVersions(uuid)
   const { version, isVersionLoading, isVersionError, mutateVersion } = useGetModelVersion(uuid, selectedVersion, true)
   const { deployments, isDeploymentsLoading, isDeploymentsError } = useGetModelDeployments(uuid)
+  const { versionAccess } = useGetVersionAccess(version?._id)
 
   const hasUploadType = useMemo(() => version !== undefined && !!version.metadata.buildOptions?.uploadType, [version])
   const sendNotification = useNotification()
 
+  // possiblyUploader stores whether an uploader could plausibly have access to privileged functions.
+  // It defaults to true, until it hears false from the network access check.
+  const possiblyUploader = useMemo(() => versionAccess?.uploader !== false, [versionAccess])
```

Comment:
Would this be easier to read if `possiblyUploader` was renamed to something like `hasUploadAccess` or `isUploader`?